### ISSUE TITLE
Shared preferences for the last typed username & Dropdown Suggestion of the same in HomeScreen EditText

### ISSUE THREAD

**Issue by Avinaash7** [ACM-VIT/gitpositive#37]
- [x] Shared Preferences which stores the previously typed username by the user.
- [x] Dropdown Suggestion for the EditText which suggests the username stored in shared preferences.

**Comment by rooparsh** [comment]
@Avinaash7 assign this to me. I will look it up by EOD today.

**Comment by Avinaash7** [comment]
Good Luck !! @rooparsh

**Comment by rooparsh** [comment]
@Avinaash7 Please check the PR [Link](https://github.com/ACM-VIT/gitpositive/pull/46) and merge it if ok.

**Comment by Avinaash7** [comment]
Merged, Good work @rooparsh !!
Happy Hacktoberfest !!

**Comment by rooparsh** [comment]
> This issue has been approved by the owner and is open to solve!
> @Avinaash7 got 5 points for this issue! 🎉

@Avinaash7 just curious, what's the concept of points. 😅

**Comment by gaganmalvi** [comment]
TL;DR https://www.instagram.com/p/CUm8zkjLMZJ/
Apart from the Hacktoberfest tees provided by Digital Ocean, we also have additional bonuses in store for our top three contributors, in our repositories for the app, web, and research domains.
